src/relay/op/contrib/ethosu/binary_elementwise.cc

```diff
@@ -128,6 +128,21 @@ struct EthosuBinaryElementwiseAttrs : public tvm::AttrsNode<EthosuBinaryElementw
 
 TVM_REGISTER_NODE_TYPE(EthosuBinaryElementwiseAttrs);
 
+bool IsScalarTensor(const Array<PrimExpr>& ifm_shape, const DataType& ifm_dtype) {
+  if (ifm_dtype != DataType::UInt(8)) {
```

Comment:
This does not qualify as not being scalar I suppose. May be worth extracting that out for clarity ?

Reply:
This will be removed in a follow up commit, so I think it would be okay to leave till then